src/bblocks_data_importers/utilities.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+
+
+def ensure_pyarrow_dtypes(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Better name might be `convert_pyarrow_dtypes` to avoid confusion about doing a check vs a conversion. Might need some error handling here

Reply:
sounds good!